docs/howto/static-files/index.txt

```diff
@@ -137,6 +137,22 @@ this by adding the following snippet to your urls.py::
     the given prefix is local (e.g. ``/media/``) and not a URL (e.g.
     ``http://media.example.com/``).
 
+Customizing the ignored pattern list
```

Comment:
If we decide to adopt this "settings on appconfigs" pattern more widely, let's add a section to somewhere like docs/ref/applications.txt where we can link to so we don't have to restate this pattern everywhere. For now, we might include this in ref/contrib/staticfiles.txt where "Don’t ignore the common private glob-style patterns" is mentioned. It feels a bit out of place in the howto.